demo/src/app/components/toast/overview/toast-overview.component.html

```diff
@@ -0,0 +1,94 @@
+<p>
+  Toasts provide feedback messages as notifications to the user.<br />
+  Goal is to mimic the push notifications available both on mobile and desktop operating systems.
+</p>
+
+<ngbd-overview-section [section]="sections['inline-usage']">
+  <p><a [routerLink]="['..', 'api']" fragment="NgbToast">NgbToast</a> component allows you to only render the corresponding markup. Use it in one of your template, and you are done. It will render a toast.</p>
+  <ngbd-code [snippet]="TOAST_INLINE_BASIC"></ngbd-code>
+  <br />
+  <p>
+    Nonetheless, with this inline technique, you must handle the toast's lifecycle yourself, i.e. it won't dissapper automagically.
+    To actually make it disappear, you can listen to the <a [routerLink]="['..', 'api']" fragment="NgbToast"><code>(hide)</code></a>
+    output and remove/destroy/hide it yourself.
+  </p>
+  <ngbd-code [snippet]="TOAST_INLINE_LIFECYCLE"></ngbd-code>
+  <br />
+  <p>Live example available <a [routerLink]="['..', 'examples']" fragment="inline" title="Declarative inline usage">here</a>.</p>
+</ngbd-overview-section>
+
+<ngbd-overview-section [section]="sections['toast-service']">
+  <p>Let's take the opportunity to demonstrate how to simply build a global toast management service.</p>
+  <ngb-alert [dismissible]="false" type="secondary">
+    <strong>TLDR;</strong>
+    You don't feel reading these long explanations? Go to the live example <a [routerLink]="['..', 'examples']" fragment="howto-global" title="Toast management service">here</a>.
+  </ngb-alert>
+
+  <p>In order to create our global toast system, 3 simple steps need to be done:<p>
+  <ol>
+    <li>Create a service <code>AppToastService</code>. Goal for this service is to act as a global storage.</li>
+    <li>Create a container component <code>&lt;app-toasts&gt;</code>, acting as the host in the application to display your toasts.
+    You could use <code>&lt;ngb-toast&gt;</code> with an <code>*ngFor</code> to read the list of toasts to display from the service.</li>
+    <li>Finally, use this container component in your application.</li>
+  </ol>
+
+  <h4>Global storage: DI to the rescue</h4>
+  <p>
+    Relying on Angular dependency injection to share some piece of logic application-wide is always a good and solid starting choice.
+  </p>
+  <p>
+    The service manages a collection of toasts. It also provides a public method to push a new toast to that same collection.
+    <ngbd-code [snippet]="APP_TOAST_SERVICE"></ngbd-code>
+  </p>
+
+  <ngb-alert [dismissible]="false" type="warning">
+    <svg:svg ngbdIcon="lightbulb" fill="currentColor" />
+    You could also create an interface to type your toast instead of using <code>any[]</code> here.
+  </ngb-alert>
+  <p>
+    Additionnally, a method to remove an existing toast from the collection is also implemented.
```

Comment:
I did that one on purpose, to avoid having big chunk of snippet.

For me rule is simple: anytime you find yourself putting a comment into a snippet it means the snippet could be split, and comment should be part of main flow of documentation.